testing/s3minio/src/main/java/org/projectnessie/minio/MinioExtension.java

```diff
@@ -51,14 +51,10 @@ public ConditionEvaluationResult evaluateExecutionCondition(ExtensionContext con
     if (OS.current() == OS.LINUX) {
       return enabled("Running on Linux");
     }
-    if (OS.current() == OS.MAC && System.getenv("CI_MAC") == null) {
-      // Disable tests on GitHub Actions
-      return enabled("Running on macOS locally");
+    if (OS.current() == OS.MAC) {
+      return enabled("Running on macOS");
     }
-    return disabled(
-        format(
-            "Disabled on %s, because it doesn't support wildcard localhost FQDNs.",
-            OS.current().name()));
+    return disabled(format("Disabled on %s", OS.current().name()));
```

Comment:
TBH I think the extension is also compatible with Windows now, but I can't really guarantee.